macros/PGbasicmacros.pl

```diff
@@ -2032,7 +2032,7 @@ sub EV3P_parser {
   my $start = ''; my %end = ('\('=>'\)','\['=>'\]');
   my @parts = split(/(``.*?``\*?|`.+?`\*?|(?:\\[()\[\]]))/s,$string);
   foreach my $part (@parts) {
-    if ($part =~ m/^(``?)(.*)\1(\*?)$/s) {
+    if ($part =~ m/^(``?)(.*)$1(\*?)$/s) {
```

Comment:
Geoff is correct, this should not have been changed.  The `\1` is used here to make sure the end quotes are the same as the beginning quotes.  `$1` doesn't work in this situation.